code/modules/research/designs/medical_designs.dm

```diff
@@ -84,6 +84,39 @@
 	build_path = /obj/item/weapon/reagent_containers/glass/beaker/noreact
 	category = list("Medical Designs")
 
+/datum/design/bluespacesyringe
+	name = "Bluespace Syringe"
+	desc = "An advanced syringe that can hold 60 units of chemicals"
+	id = "bluespacesyringe"
+	req_tech = list("bluespace" = 1, "materials" = 2)
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 500, MAT_PLASMA = 1000, MAT_DIAMOND = 500)
+	reliability = 76
+	build_path = /obj/item/weapon/reagent_containers/syringe/bluespace
+	category = list("Medical Designs", "Bluespace Designs")
+
+/datum/design/noreactsyringe
+	name = "Cryo Syringe"
+	desc = "An advanced syringe that stops reagents inside from reacting. It can hold up to 20 units."
+	id = "noreactsyringe"
+	req_tech = list("materials" = 2)
+	build_type = PROTOLATHE
```

Comment:
might wanna give this one weapons tech as well, as it's intended to be weaponized.